packages/table/src/quadrants/tableQuadrantStack.tsx

```diff
@@ -841,27 +884,31 @@ export class TableQuadrantStack extends AbstractComponent<ITableQuadrantStackPro
         }
     }
 
+    private maybeIncreaseToDefaultColumnHeaderHeight(height: number) {
+        return height <= QUADRANT_MIN_SIZE ? DEFAULT_COLUMN_HEADER_HEIGHT : height;
```

Comment:
FYI: I tried and failed to remove the need for this `QUADRANT_MIN_SIZE` thing. Can revisit in a bigger DOM refactor.